### PR TITLE
chore: translations for login.properties

### PR DESCRIPTION
**Pull request by ui-okta**
## Description
Auto copy of latest translations by Okta-smartling script.
### Bacon task details
- Task ID: d356b1284ac04f0498257d9c336edc00 
- Requested by: evelyn.teo@okta.com
- Resolves: [https://oktainc.atlassian.net/browse/OKTA-419360; https://oktainc.atlassian.net/browse/OKTA-419362](https://oktainc.atlassian.net/browse/https://oktainc.atlassian.net/browse/OKTA-419360; https://oktainc.atlassian.net/browse/OKTA-419362)
## Notification sent to
@okta/nxt @haishengwu-okta @nikhilvenkatraman-okta @aarongranick-okta @mauriciocastillosilva-okta

### CODE REVIEW

File: packages/@okta/i18n/src/properties/login_da.properties [hunk 1]
```diff
@@ -930,6 +940,14 @@ oie.symantecVip.challenge.title = Bekr\u00E6ft med {0}
 oie.symantecVip.challenge.description = Indtast den genererede sikkerhedskode fra {0}-appen
 oie.symantecVip.verify.passcode.label = Angiv sikkerhedskode
 
+## YubiKey Authenticator
+oie.yubikey.label = YubiKey Authenticator
```
Comment (jmelberg-okta) [inline packages/@okta/i18n/src/properties/login_da.properties:944]: Shouldn't "Authenticator" need to be translated here?

File: packages/@okta/i18n/src/properties/login_cs.properties [hunk 2]
```diff
@@ -533,10 +533,10 @@ primaryauth.submit = Prihl\u00E1sit
 primaryauth.newUser.tooltip=Toto je va\u0161e prvn\u00ED pripojen\u00ED k {0} z tohoto prohl\u00ED\u017Eece
 primaryauth.newUser.tooltip.close=Zavr\u00EDt
 oktaVerify.description = Pro p\u0159\u00EDstup k tomuto zdroji vy\u017Eaduje va\u0161e organizace p\u0159ihl\u00E1\u0161en\u00ED pomoc\u00ED va\u0161eho za\u0159\u00EDzen\u00ED.
-oktaVerify.button = P\u0159ihlaste se pomoc\u00ED Okta Verify na tomto za\u0159\u00EDzen\u00ED
+oktaVerify.button = P\u0159ihl\u00E1sit se pomoc\u00ED Okta FastPass
 
 # IDP Discovery
-idpDiscovery.email.placeholder = Email
+idpDiscovery.email.placeholder = E-mail
```
Comment (jmelberg-okta) [inline packages/@okta/i18n/src/properties/login_cs.properties:539]: We're starting to converge on using the hyphenated spelling for this language, but there appears to be a few [instances](https://github.com/okta/okta-signin-widget/blob/c4f0536b85b316cf7f38fdc3fdf2b6b1bd5b49cc/packages/%40okta/i18n/src/properties/login_cs.properties#L665) where we aren't.

File: packages/@okta/i18n/src/properties/login_fi.properties [hunk 3]
```diff
@@ -533,7 +533,7 @@ primaryauth.submit = Kirjaudu sis\u00E4\u00E4n
 primaryauth.newUser.tooltip=T\u00E4m\u00E4 on ensimm\u00E4inen {0} -yhteydenottosi t\u00E4lt\u00E4 selaimelta
 primaryauth.newUser.tooltip.close=Sulje
 oktaVerify.description = Organisaatiosi vaatii sinua kirjautumaan sis\u00E4\u00E4n laitteellasi, ett\u00E4 voit k\u00E4ytt\u00E4\u00E4 t\u00E4t\u00E4 resurssia.
-oktaVerify.button = Kirjaudu sis\u00E4\u00E4n Okta Verifyll\u00E4 t\u00E4ll\u00E4 laitteella
+oktaVerify.button = Kirjaudu sis\u00E4\u00E4n Okta FastPassilla
```
Comment (jmelberg-okta) [inline packages/@okta/i18n/src/properties/login_fi.properties:536]: Double checking - isn't this a reserved product name? `Fastpass`

File: packages/@okta/i18n/src/properties/login_fi.properties [hunk 4]
```diff
@@ -930,6 +940,14 @@ oie.symantecVip.challenge.title = Todenna seuraavalla tavalla: {0}
 oie.symantecVip.challenge.description = Anna {0} -sovelluksessa luotu turvakoodi.
 oie.symantecVip.verify.passcode.label = Sy\u00F6t\u00E4 turvakoodi
 
+## YubiKey Authenticator
+oie.yubikey.label = YubiKey Authenticator
```
Comment (jmelberg-okta) [inline packages/@okta/i18n/src/properties/login_fi.properties:944]: Interesting seeing this translated in only a few languages. Let's confirm that this is correct before merging.

**Review by jmelberg-okta — APPROVED** [review]
Couple of verification questions for @evelynteo-okta. Assuming everything is valid, this is good to go!

File: packages/@okta/i18n/src/properties/login_fi.properties (hunk 3 above)
Comment (evelynteo-okta) [reply packages/@okta/i18n/src/properties/login_fi.properties:536]: This is the Finnish language handling of proper nouns. It's part of the grammatical construct.

File: packages/@okta/i18n/src/properties/login_ru.properties [hunk 5]
```diff
@@ -8,7 +8,7 @@ signin = \u0412\u043E\u0439\u0442\u0438 \u0432 \u0441\u0438\u0441\u0442\u0435\u0
...
 remember = \u0417\u0430\u043F\u043E\u043C\u043D\u0438\u0442\u044C \u043C\u0435\u043D\u044F
 rememberDevice = \u0414\u043E\u0432\u0435\u0440\u044F\u0442\u044C \u044D\u0442\u043E\u043C\u0443 \u0443\u0441\u0442\u0440\u043E\u0439\u0441\u0442\u0432\u0443
-rememberDevice.timebased = \u041D\u0435 \u043F\u0440\u043E\u0432\u043E\u0434\u0438\u0442\u044C \u0430\u0443\u0442\u0435\u043D\u0442\u0438\u0444\u0438\u043A\u0430\u0446\u0438\u044E {0} \u043D\u0430 \u044D\u0442\u043E\u043C \u0443\u0441\u0442\u0440\u043E\u0439\u0441\u0442\u0432\u0435 \u0432 \u0442\u0435\u0447\u0435\u043D\u0438\u0435 \u0441\u043B\u0435\u0434\u0443\u044E\u0449\u0438\u0445 {0}
+rememberDevice.timebased = \u041D\u0435 \u043E\u0442\u043F\u0440\u0430\u0432\u043B\u044F\u0439\u0442\u0435 \u043C\u043D\u0435 \u0437\u0430\u043F\u0440\u043E\u0441/\u043E\u0442\u0432\u0435\u0442 \u043D\u0430 \u0430\u0443\u0442\u0435\u043D\u0442\u0438\u0444\u0438\u043A\u0430\u0446\u0438\u044E \u0432 \u0442\u0435\u0447\u0435\u043D\u0438\u0435 \u0441\u043B\u0435\u0434\u0443\u044E\u0449\u0438\u0445 {0} \u043C\u0438\u043D\u0443\u0442 \u043D\u0430 \u044D\u0442\u043E\u043C \u0443\u0441\u0442\u0440\u043E\u0439\u0441\u0442\u0432\u0435.
```
Comment (evelynteo-okta) [inline packages/@okta/i18n/src/properties/login_ru.properties:11]: @keithchin-okta https://oktainc.atlassian.net/browse/OKTA-414525
I tagged the bacon task with the JIRA but it didn't show up. Here is the Russian fix.

File: packages/@okta/i18n/src/properties/login_da.properties (hunk 1 above)
Comment (nikhilvenkatraman-okta) [reply packages/@okta/i18n/src/properties/login_da.properties:944]: I have the same question. It seems to be translated in certain languages (nb, sv). Not sure if this is a language specific thing.

**Review by nikhilvenkatraman-okta — APPROVED** [review]
Is this the monthly translation request for August ? or is this a mid-month one off request ?

File: packages/@okta/i18n/src/properties/login_es.properties [hunk 6]
```diff
@@ -856,17 +866,17 @@ oie.okta_verify.totp.enterCodeText = Introduzca el c\u00F3digo de la aplicaci\u0
 oie.okta_verify.push.title = Obtener una notificaci\u00F3n push
 oie.okta_verify.push.sent = Se ha enviado una notificaci\u00F3n push
 oie.okta_verify.push.resend = Reenviar la notificaci\u00F3n push
-oie.okta_verify.signed_nonce.label = Utilizar Okta Verify en este dispositivo
+oie.okta_verify.signed_nonce.label = Use Okta\u00A0FastPass.
```
Comment (helenchen-okta) [inline packages/@okta/i18n/src/properties/login_es.properties:869]: Shouldn't this be "Utilizar Okta\u00A0FastPass"? Also why is a period added?

Comment (evelynteo-okta) [reply packages/@okta/i18n/src/properties/login_es.properties:869]: @helenchen-okta "Use" is correct. It's the conjugation of the verb "usar" (to use) in Spanish. See: https://www.spanishdict.com/conjugate/usar

Period is the Spanish language convention. It's uncommon to see sentences without punctuation at the end.

File: packages/@okta/i18n/src/properties/login_da.properties (hunk 1 above)
Comment (evelynteo-okta) [reply packages/@okta/i18n/src/properties/login_da.properties:944]: The English loan word "authenticator" entered the technical term lexicon in these languages. This is expected.

File: packages/@okta/i18n/src/properties/login_fi.properties (hunk 4 above)
Comment (evelynteo-okta) [reply packages/@okta/i18n/src/properties/login_fi.properties:944]: The English loan word "authenticator" entered the technical term lexicon in these languages. This is expected.

File: packages/@okta/i18n/src/properties/login_cs.properties (hunk 2 above)
Comment (evelynteo-okta) [reply packages/@okta/i18n/src/properties/login_cs.properties:539]: I'll have the team scrub the compound construct in the next delivery. It's not a show-stopping change.

**Pull request closed without merging**